docs/dictionary/control_st/next-repeat.lcdoc

```diff
@@ -24,12 +24,14 @@ anywhere inside a <repeat> <control structure>.
 The <next repeat> <control structure> skips the rest of the current
 <iteration>, and continues to the next <iteration>. The following
 example performs the loop action only if the <current card|current
-card's> name contains the letter "e" :
-
-    repeat for the number of cards of this stack
-        go to next card
-        if "e" is in the short name of this card then next repeat
-        put the short name of this card & return after myCardsList
+card's> name does not begin with the character ".":
```

Comment:
This should be ...performs the loop action only if the folder name does not begin with the character "."